CSharp/Library/Microsoft.Bot.Connector.Shared/OAuthAPI/OAuthApi.cs

```diff
@@ -297,7 +291,7 @@ public Task<TokenResponse> GetUserTokenAsync(string userId, string connectionNam
             return String.Empty;
         }
 
-        public async Task SendEmulateOAuthCardsAsync(bool emulateOAuthCards)
+        public async Task SendEmulateOAuthCardsAsync(string emulatorUri, bool emulateOAuthCards)
```

Comment:
>SendEmulateOAuthCardsAsync(string emulatorUri, bool emulateOAuthCards) [](start = 26, length = 70)

isn't this a breaking API change?

Reply:
Yes, good point. Here's how I fixed this: I reverted this part in favor of making a new oauth client that points just at the emulator. See latest changes.